rever.xsh

```diff
@@ -0,0 +1,33 @@
+$PROJECT = $GITHUB_ORG = $GITHUB_REPO = 'xonsh'
+$WEBSITE_URL = 'http://xon.sh'
+$ACTIVITIES = ['version_bump', 'changelog', 'pytest',
+               'sphinx', 'tag', 'pypi', 'conda_forge', 'ghpages', 'ghrelease'
+               ]
+
+$VERSION_BUMP_PATTERNS = [
+    ('.appveyor.yml', 'version:.*', 'version: $VERSION.{build}'),
+    ('xonsh/__init__.py', '__version__\s*=.*', "__version__ = '$VERSION'"),
+    ]
+$CHANGELOG_FILENAME = 'CHANGELOG.rst'
+$CHANGELOG_TEMPLATE = 'TEMPLATE.rst'
+
+#$PYTEST_COMMAND = "rm -r .cache/ __pycache__/ */__pycache__ */*/__pycache__ && pytest"
+
+$TAG_REMOTE = 'git@github.com:xonsh/xonsh.git'
+$TAG_TARGET = 'master'
+
+$GHPAGES_REPO = 'git@github.com:scopatz/xonsh-docs.git'
```

Comment:
Should this eventually go under xonsh the org?

Reply:
Yes. It mostly is, but we haven't switched the CNAME file over yet.

Reply:
ah ok